checkov/terraform/plan_runner.py

```diff
@@ -31,6 +32,12 @@
     "CKV_GCP_82",
 }
 
+RESOURCE_ATTRIBUTES_TO_OMIT = {
+    'azurerm_key_vault_secret': 'value',
+    'aws_secretsmanager_secret_version': 'secret_string',
+    'google_kms_secret_ciphertext': 'plaintext'
```

Comment:
should we also add SSM parameter to the list or in a follow up PR? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_parameter#encrypted-string-using-default-ssm-kms-key as I remember they are saved as plain text in the plan file, but I could be wrong.

Reply:
Good catch, so is that true also for `aws_db_instance`: `password` in plan files? @gruebel

Reply:
good question, yeah it is `All arguments including the username and password will be stored in the raw state as plain-text.`

Reply:
lets create a new GitHub issue with example resources, which can omit secrets in the TF plan file